src/components/ui/BuyButton/BuyButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Button as UIButton } from '@faststore/ui'
+import type { ButtonProps } from '@faststore/ui'
+import { ShoppingCart } from 'phosphor-react'
```

Comment:
It occurs to me that we may need to have a unique source to retrieve these icons. It has two main reasons:

1) If we decide to change the icon, we will need to change it everywhere it's used. The `ShoppingCart` is already being used at the [CartButton component](https://github.com/vtex-sites/base.store/pull/159), for example.

2) `ShoppingCart` is misleading. It would be good to have some indication that it's an Icon.

I don't think this is immediately necessary. We may tackle it later if we decide to do so. What do you think?

Reply:
hmm! I think it might be a good idea! We can name it with `Icon` suffix it will be more clear ex: `ShoppingCartIcon`

Reply:
well noticed, dan. I will fix it right away